### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by Ukaypromise**
#### For this project, relational database was used to create the initial data structure for a vet clinic. Various animal information was stored in the tables using SQL and then they were queried.

1. Find all animals whose name ends in "mon".
correction

2. List the name of all animals born between 2016 and 2019.
date 2016-2019
3. List the name of all animals that are neutered and have less than 3 escape attempts.
neutered escape

4. List the date of birth of all animals named either "Agumon" or "Pikachu".
name augummm

5. List name and escape attempts of animals that weigh more than 10.5kg
name escape

6. Find all animals that are neutered.
animals neutered

7. Find all animals not named Gabumon.
name not gabmon

8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
weight between

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name like '%mon%'
```
Comment (omar-muhamad) [inline queries.sql]: - [x] This query suppose to get all animals whose name ends in `mon` but this query will get any animal that his name contains `mon`. Please fix that.
[Hint] You can do it like that:
```sql
SELECT * FROM animals WHERE name like '%mon'
```

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Ukaypromise,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (Ukaypromise) [reply queries.sql]: Fix !!! @Omar-Muhamad

**Review by gemmen29 — APPROVED** [review]
Hi @Ukaypromise, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**